submitted_models/ctu_cras_norlab_spot_sensor_config_1/model.sdf

```diff
@@ -190,7 +193,7 @@
       <light name="light_left" type="spot">
         <pose>0.275 0.1 0.13 1.5707963267948966 -1.5707963267948966 0</pose>
         <cast_shadows>1</cast_shadows>
-        <attenuation>
+        <attenuation>ign sdf -p
```

Comment:
Is this line intended?

Reply:
Fixed in https://github.com/osrf/subt/pull/953/commits/dd1bc50e8fed275918072026f6179aa274882a50